app/javascript/courses/curricula/components/CoursesCurriculum.re

```diff
@@ -312,12 +313,19 @@ let make =
 
       {
         selectedLevelId:
-          switch (targetLevelId, levelZero) {
-          | (Some(targetLevelId), Some(levelZero)) =>
+          switch (preview, targetLevelId, levelZero) {
+          | (true, None, None) =>
+            let levelOne =
+              levels |> List.find_opt(level => level |> Level.number == 1);
```

Comment:
This should be replaced with `ListUtils.unsafeFind`

Reply:
Fixed in dc4cd159c1919ed890fb1258b29b790329697962